guides/common/images/mcp-architecture-2.mmd

```diff
@@ -0,0 +1,38 @@
+---
+config:
+    theme: default
+---
+architecture-beta
+    service user(mdi:account)[User 1]
+
+    group user-system(mdi:laptop)[User system 1]
+        group mcp-host[AI application 1] in user-system
+            service mcp-client(mdi:checkbox-intermediate)[MCP client 1] in mcp-host
+        service mcp-server(mdi:checkbox-intermediate)[MCP server 1] in user-system
+
+    user:R <--> L:mcp-client{group}
+    mcp-client:R <--> L:mcp-server
```

Comment:
I always see these lines as the initiator and responses are IMHO implied. Should it be:
```suggestion
    user:R --> L:mcp-client{group}
    mcp-client:R --> L:mcp-server
```

If so, this is recurring item